### PR TITLE
libsonata-report: update: add module.

### PR DESCRIPTION
**Pull request by jorblancoa**
- Add module for the reports and spikes converter

### CODE REVIEW

File: bluebrain/deployment/environments/applications_libraries.yaml [hunk 1]
```diff
@@ -9,6 +9,7 @@ spack:
           - gmsh
           - hdf5+mpi
           - highfive+mpi
+          - libsonata-report
```
Comment (pramodk) [inline bluebrain/deployment/environments/applications_libraries.yaml]: @jorblancoa : do we need to expose that as a module on BB5? i.e. do you expect our scientists to use it?

Its ok to install. Just wondering if it's going to be used via module. (Or some of us as a devs, just do `spack load`)

Comment (jorblancoa) [reply bluebrain/deployment/environments/applications_libraries.yaml]: makes sense, so maybe leave it only in specs?

Comment (pramodk) [reply bluebrain/deployment/environments/applications_libraries.yaml]: that's what I am thinking 👍

**Review by pramodk — APPROVED** [review]
LGTM

**Comment by pramodk** [comment]
@jorblancoa : enable auto squash merge 😀

**Pull request merged**